packages/keymaster/src/types.ts

```diff
@@ -29,9 +29,15 @@ export interface WalletFile {
     names?: Record<string, string>
 }
 
-export type StoredWallet = WalletFile | EncryptedWallet | null
+export type WrappedWallet = WalletFile | null;
+export type StoredWallet = WrappedWallet | EncryptedWallet;
 
 export interface WalletBase {
     saveWallet(wallet: StoredWallet, overwrite?: boolean): Promise<boolean>
     loadWallet(): Promise<StoredWallet>
 }
+
+export interface WalletWrapper {
```

Comment:
Not sure why this is necessary. The point of the wallet wrappers is to expose exactly the same interface as the wrapped wallet?

Reply:
WalletWrapper is for the encryption and cache wrapper as it returns WrappedWallet which does not include an encrypted type.